### ISSUE TITLE
Prevent click on the step

### ISSUE THREAD

**Issue by entwicklerRepo** [adi518/vue-stepper-component#5]
Can I prevent click on the step somehow? I don't want to allow moving to the next step (unless user let's say filled form)

**Comment by adi518** [comment]
Hi, I'm rewriting this component and the new version will allow the consumer much more control.

**Comment by entwicklerRepo** [comment]
@adi518 oh nice! is there any estimates?

**Comment by theGrower** [comment]
Any progress on how to prevent clicking the steps?

**Comment by theGrower** [comment]
temporary CSS workaround here: 
```
     /* prevent click events on all steppers */
div.v-stepper > div.v-stepper-root > div.v-step {
    pointer-events: none;
}
```

**Comment by adi518** [comment]
Sorry to disappoint, my plans were slightly hindered by real life. I don't like to leave things uncompleted and I do want to keep this little project maintained, so I will take care of it once things settle down a little.

**Comment by theGrower** [comment]
No worries! Life and work happens.

**Comment by justinbeatz** [comment]
Any update on this? Also, do you have the WIP in another branch? Maybe you could uploaded and get some help from the community.

**Comment by theGrower** [comment]
> 
> 
> Any update on this? Also, do you have the WIP in another branch? Maybe you could uploaded and get some help from the community.

We ended up writing our own stepper and have abandoned this one. Good luck @justinbeatz

**Comment by adi518** [comment]
@theGrower Did you open-source it? I'm looking to get back to this and modernize it for Vue 2/3, unless there's a better one out there.

**Comment by theGrower** [comment]
@adi518 No, we did not open-source it. I'm sorry. I'm also no longer at that company to be able to access the code base and discuss making it open source.

**Comment by adi518** [comment]
That was a quick reply, thanks. I looked at [`awesome-vue`](https://github.com/vuejs/awesome-vue#wizard) and couldn't find new alternatives. This means it's probably worth reviving this one.

**Comment by theGrower** [comment]
@adi518 Honestly, it wasn't that much work to build our own with the exact functionality we wanted. I think the other dev, granted she's pretty darn good, had it build in an afternoon. Most of the framework we needed was there in HTML5, ECMAScript 6, and Vue.js. A little custom CSS and, BOOM, it was up and running. 
I'd say it's worth giving it a go. 
If I remember correctly, it was essentially just re-styling tabs maybe? That was a long time ago.

**Comment by adi518** [comment]
Which developer are you talking about? btw, it is generally not wise to build a component from scratch. It's better to fork and add the functionality you want unless it deviates significantly from the original functionality. That way you can also contribute back to the original project, be it an individual or software company. This component has sophisticated logic for customizing it to a large extent, something many other components do not offer. I can also say that most of the issues that people start are related to insufficient knowledge with Vue.js and using its scoped slot feature.

**Comment by theGrower** [comment]
Sorry, the Dev that built out our own stepper at my last employer. 
I agree with you; however, to fork this project, fix what we needed, and maintain it would have been a lot more work than we had the ability to take on. By all means, I support you in this effort. Link me the project, I'd like to follow its progress.

**Comment by theGrower** [comment]
Oh, also, I'm just realize which topic this is. I was thinking of a different topic... hell, it may have been a different stepper we tried. I'm sorry. You are the maintainer as well. 
My bad. I do remember the CSS I wrote above and it worked, and then we wrote our own. I'm completely out of context in my thoughts.
